### PR TITLE
Adding TTL to the json accessToken response

### PR DESCRIPTION
**Pull request by smith64fx**
In a Single Page Application you want to know the TTL so you can set the right cookie expiration date.

#### Related issues

- None

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (superkhau) [inline .vscode/settings.json]: Remove this file please.

File: lib/passport-configurator.js [hunk 2]
```diff
@@ -512,10 +510,7 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
         } else {
           if (info && info.accessToken) {
             if (!!options.json) {
-              return res.json({
-                'access_token': info.accessToken.id,
-                userId: user.id,
```
Comment (superkhau) [inline lib/passport-configurator.js]: Why remove the userId?

**Comment by superkhau** [comment]
@smith64fx Can you add a test to verify the TTL is returned?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by raymondfeng** [comment]
@smith64fx Thank you for the contribution. Can you address @superkhau's comments?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**